src/server/pachyderm_test.go

```diff
@@ -2200,3 +2241,17 @@ func restartAll(t *testing.T) {
 	}
 	waitForReadiness(t)
 }
+
+func restartOne(t *testing.T) {
+	k := getKubeClient(t)
+	podsInterface := k.Pods(api.NamespaceDefault)
+	labelSelector, err := labels.Parse("suite=pachyderm")
```

Comment:
This could lead to restarting the rethink or etcd pod which I don't think tests what we want. You should do `app=pachd` instead.